plugins/meta/portmap/main.go

```diff
@@ -164,7 +164,10 @@ func parseConfig(stdin []byte, ifName string) (*PortMapConf, error) {
 			}
 
 			// Skip known non-sandbox interfaces
-			intIdx := ip.Interface
+			if ip.Interface == nil {
```

Comment:
I still don't think that's right.

Basically, I was trying to make the plugin as flexible as possible (though this might not be a good idea). In other words, it tries to skip an IP if it's definitely sure that it's not on the container. In the upgrade instructions, for example, we say that the Interface can be set to -1 if desired.

If you think the plugin should be stricter, maybe so. But that should probably be a separate PR.